openshift/servicemesh/operator-install.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: operators.coreos.com/v1
```

Comment:
do we really need all of this?

here is what we apply in eventing:
https://github.com/openshift/knative-eventing/blob/release-v0.8.0/openshift/serverless/operator-install.yaml

Reply:
thanks for the references and review, updated to use remote addresses and removed all the vendored files.